### ISSUE TITLE
Discussion: Move to Ctrl+C instead of `Hit any key to exit`

### ISSUE THREAD

**Issue by ghuntley** [statiqdev/Statiq#534]
When copy and pasting things from terminal I often press enter to copy the contents of the terminal to the clipboard. If I press enter one too many times Wyam exists. Very annoying. Any compelling reason to stay with `Console.ReadLine()` vs moving to the more popular convention of  `Ctrl+C` to exit.

**Comment by daveaglick** [comment]
Nope, no reason at all not to switch. Another easy one I can add for the next release.

**Comment by daveaglick** [comment]
This will go out with the next release. Turns out it wasn't *totally* straightforward. I discovered that at least in one case (mine), the Ctrl-C comes through as `(ConsoleKey)3` (which isn't even a valid `ConsoleKey` enum value). No idea why. And the `Console.CancelKeyPress` event allows the Ctrl-C combo to bubble up to the calling cmd/process after execution ends (even with the `.Cancel` flag). So anyway, learned a lot about cancel key commands in the console :smile:

![2017-06-29_18h11_08](https://user-images.githubusercontent.com/1020407/27713028-a733b302-5cf7-11e7-8fdb-30301db9c873.png)
